target/linux/rockchip/patches-6.6/401-2-arm64-dts-rockchip-rk3566-Nanopi-R3S-update-LED.patch

```diff
@@ -0,0 +1,14 @@
+--- a/arch/arm64/boot/dts/rockchip/rk3566-nanopi-r3s.dts
```

Comment:
please renumber this to 128, i.e. `target/linux/rockchip/patches-6.6/128-arm64-dts-rockchip-rk3566-Nanopi-R3S-update-LED.patch`

Reply:
Thanks, done